android-vjestina-homework-movie-app-data-mock/app/src/main/java/agency/five/codebase/android/movieapp/ui/favourites/FavoritesScreen.kt

```diff
@@ -0,0 +1,103 @@
+package agency.five.codebase.android.movieapp.ui.favourites
+
+import agency.five.codebase.android.movieapp.R
+import agency.five.codebase.android.movieapp.mock.MoviesMock.getMoviesList
+import agency.five.codebase.android.movieapp.ui.component.MovieCard
+import agency.five.codebase.android.movieapp.ui.component.MovieCardViewState
+import agency.five.codebase.android.movieapp.ui.favourites.mapper.FavoritesMapperImpl
+import agency.five.codebase.android.movieapp.ui.favourites.mapper.IFavoritesMapper
+import agency.five.codebase.android.movieapp.ui.theme.MovieAppTheme
+import agency.five.codebase.android.movieapp.ui.theme.spacing
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.grid.GridCells
+import androidx.compose.foundation.lazy.grid.GridItemSpan
+import androidx.compose.foundation.lazy.grid.LazyVerticalGrid
+import androidx.compose.foundation.lazy.grid.items
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.sp
+
+
+private val favoritesMapper: IFavoritesMapper = FavoritesMapperImpl()
+
+val favoritesViewState = favoritesMapper.toFavoritesState(getMoviesList().filter { it.isFavorite })
+
+@Composable
+fun FavoritesRoute(
+    onNavigateToMovieDetails: (FavoritesMovieViewState) -> Unit
+) {
+    val favoritesViewState by remember { mutableStateOf(favoritesViewState) }
+
+    FavoritesScreen(
+        favoritesViewState = favoritesViewState,
+        onNavigateToMovieDetails = onNavigateToMovieDetails
+    )
+}
+
+@Composable
+fun FavoritesScreen(
+    modifier: Modifier = Modifier,
+    favoritesViewState: FavoritesViewState,
+    onNavigateToMovieDetails: (FavoritesMovieViewState) -> Unit
+) {
+    Column(
+        modifier = modifier
+            .fillMaxSize()
+    ) {
```

Comment:
Why do we need this column? 
Can we use the `LazyVerticalGrid` as a root layout here?

Reply:
Yes, it's redundant.